### PR TITLE
add accessor for is_server

### PR DESCRIPTION
**Pull request by ehaydenr**

### CODE REVIEW

**Review by LPardue — DISMISSED** [review]

File: quiche/src/ffi.rs [hunk 1]
```diff
@@ -1211,6 +1211,11 @@ pub extern fn quiche_conn_path_stats(
     0
 }
 
+#[no_mangle]
+pub extern fn is_server(conn: &Connection) -> bool {
```
Comment (LPardue) [inline quiche/src/ffi.rs:1215]: also need to define this in quiche.h

File: quiche/src/lib.rs [hunk 2]
```diff
@@ -6260,6 +6260,11 @@ impl Connection {
         self.paths.iter().map(|(_, p)| p.stats())
     }
 
+    /// Returns whether or not this is a server-side connection
+    pub fn is_server(&self) -> bool {
+        return self.is_server;
```
[the remark this answers is not in the document]
Comment (ehaydenr) [reply quiche/src/lib.rs]: oof I must still be jet lagged... fixed thank you

**Review by LPardue — DISMISSED** [review]

File: quiche/src/lib.rs [hunk 3]
```diff
@@ -6260,6 +6260,11 @@ impl Connection {
         self.paths.iter().map(|(_, p)| p.stats())
     }
 
+    /// Returns whether or not this is a server-side connection
```
Comment (LPardue) [inline quiche/src/lib.rs:6263]:
```suggestion
    /// Returns whether or not this is a server-side connection.
```

File: quiche/include/quiche.h [hunk 4]
```diff
@@ -625,6 +625,8 @@ typedef struct {
 // field of `quiche_stats`).
 int quiche_conn_path_stats(const quiche_conn *conn, size_t idx, quiche_path_stats *out);
 
+bool quiche_conn_is_server(const quiche_conn *conn);
```
Comment (LPardue) [inline quiche/include/quiche.h:628]:
```suggestion
// Returns whether or not this is a server-side connection.
bool quiche_conn_is_server(const quiche_conn *conn);
```

**Review by LPardue — APPROVED** [review]

**Comment by LPardue** [comment]
merged as https://github.com/cloudflare/quiche/commit/b3c73e3dc64e7fa7360696100e5cba6d51547ec3 thanks!

**Pull request merged**